core/store/src/main/java/mil/nga/giat/geowave/core/store/GeoWaveStoreFinder.java

```diff
@@ -111,9 +110,7 @@ public static SecondaryIndexDataStore createSecondaryIndexDataStore(
 	private static List<String> getMissingRequiredOptions(
 			final StoreFactoryFamilySpi factory,
 			final Map<String, String> configOptions ) {
-		final ConfigOption[] options = ConfigUtils.createConfigOptionsFromJCommander(factory
-				.getDataStoreFactory()
-				.createOptionsInstance());
+		final ConfigOption[] options = ConfigUtils.createConfigOptionsFromJCommander(factory.getDataStoreFactory().createOptionsInstance());
```

Comment:
are the new format changes matching running maven builds (if not the next commit will end up reformatted this again)?